torch/_inductor/ir.py

```diff
@@ -6704,6 +6704,72 @@ def create(
         )
 
 
+class WeightOnlyQuantizedMatMulInt8(ExternKernelAlloc):
```

Comment:
Why cant we just use the default ExternKernel here ? this isnt doing anything special in the lowering

Reply:
Thanks, you are right. I have moved the kernel to fallback.